opm/simulators/aquifers/AquiferCarterTracy.hpp

```diff
@@ -87,94 +90,7 @@ class AquiferCarterTracy : public AquiferInterface<TypeTag>
-            this->cell_depth_.at(idx) = this->ebos_simulator_.vanguard().cellCenterDepth(cell_index);
-        }
-        // get default areas for all intersections
-        const auto& gridView = this->ebos_simulator_.vanguard().gridView();
-        ElementMapper elemMapper(gridView, Dune::mcmgElementLayout());
-        auto elemIt = gridView.template begin</*codim=*/ 0>();
-        const auto& elemEndIt = gridView.template end</*codim=*/ 0>();
-        for (; elemIt != elemEndIt; ++elemIt) {
-            const auto& elem = *elemIt;
-            unsigned cell_index = elemMapper.index(elem);
-            int idx = this->cellToConnectionIdx_[cell_index];
-
-            // only deal with connections given by the aquifer
-            if( idx < 0)
-                continue;
-
-            auto isIt = gridView.ibegin(elem);
-            const auto& isEndIt = gridView.iend(elem);
-            for (; isIt != isEndIt; ++ isIt) {
-                // store intersection, this might be costly
-                const auto& intersection = *isIt;
-
-                // only deal with grid boundaries
-                if (!intersection.boundary())
-                    continue;
-
-                int insideFaceIdx  = intersection.indexInInside();
-                switch (insideFaceIdx) {
-                case 0:
-                    faceDirection = Opm::FaceDir::XMinus;
-                    break;
-                case 1:
-                    faceDirection = Opm::FaceDir::XPlus;
-                    break;
-                case 2:
-                    faceDirection = Opm::FaceDir::YMinus;
-                    break;
-                case 3:
-                    faceDirection = Opm::FaceDir::YPlus;
-                    break;
-                case 4:
-                    faceDirection = Opm::FaceDir::ZMinus;
-                    break;
-                case 5:
-                    faceDirection = Opm::FaceDir::ZPlus;
-                    break;
-                default:
-                    OPM_THROW(Opm::NumericalIssue,
-                              "Initialization of Aquifer Carter Tracy problem. Make sure faceTag is correctly defined");
-                }
-
-                if (faceDirection == this->connections_[idx].face_dir) {
-                    this->faceArea_connected_[idx] = this->getFaceArea(intersection, idx);
-                    denom_face_areas += (this->connections_[idx].influx_mult * this->faceArea_connected_.at(idx));
-                }
-            }
-        }
-
-        const double eps_sqrt = std::sqrt(std::numeric_limits<double>::epsilon());
-        for (size_t idx = 0; idx < this->size(); ++idx) {
-            this->alphai_.at(idx) = (denom_face_areas < eps_sqrt)
-                ? // Prevent no connection NaNs due to division by zero
-                0.
-                : (this->connections_[idx].influx_mult * this->faceArea_connected_.at(idx)) / denom_face_areas;
-        }
-    }
+    Scalar fluxValue; // value of flux
```

Comment:
Please add trailing underscore in line with other class variables here. Also, the `this->` convention is used for other members, so best to apply it to fluxValue as well.

Reply:
done.